src/Accordion/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { string, bool, element, oneOfType, func } from 'prop-types';
+import classNames from 'classnames';
+import { Icon } from 'watson-react-components';
+
+Accordion.propTypes = {
```

Comment:
i didn't think you could do static proptypes when you have a functional component?